tests/tailer/access/tests.csv

```diff
@@ -0,0 +1,15 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,.rdb.subtables:(),1,,"Clear subscription tables"
+before,0,0,q,.rdb.subscribe[],1,,"Subscribe to all data from stp"
+before,0,0,q,system "sleep 5",1,,"Wait for tables to populate"
+before,0,0,q,rdbh:gethandle[`rdb1],1,,"Get rdb handle"
+before,0,0,q,stph:gethandle[`stp1],1,,"Get stp handle"
+true,0,0,q,.ds.datastripe=1b,1,,"Check datastriping is set to 1"
+true,0,0,q,2=count .wdb.access,1,,"Check length of access table in wdb"
+true,0,0,q,rdbh"2=count .rdb.access",1,,"Check length of access table in rdb"
+run,0,0,q,stph".stplg.endofperiod[.z.d+11:50;.z.d+12:00;.stplg.endofdaydata]",1,,"Run end of period function from stp"
+true,0,0,q,.wdb.access[`quote][`end]=.z.d+12:00,1,,"check end time in wdb access table"
+true,0,0,q,rdbh".rdb.access[`quote][`start]>=.z.d+11:20",1,,"check start time in rdb access table"
```

Comment:
Are these checks on the start and end time on rdb and wdb checked from the data values in the accesstables?